rfcs/52-unstable_features.md

```diff
@@ -0,0 +1,84 @@
+
+The feature gate should follow the following convention for its name: `unstable-<#tracking-issue-number>-<feature-name>`.
+
+An existing PR can be retroffited in this process on the suggestion of Bevy community members, and if the PR author agrees.
+
+The RFCs and the unstable features are closely related. An RFC can be implemented as an unstable feature to help resolve issues that need more experimentation, or split a large implementation. An unstable feature could require an RFC to reach stabilization. For example:
+- RFC to Unstable Feature
+    - Open a RFC, reaching a design
+    - Implement part of the RFC as an unstable feature
+    - Feedback into the RFC with the gained knowledge of implementation and usage
+- Unstable Feature to RFC
+    - Open an tracking issue, start implementing it
+    - Part of the unstable feature is larger than expected with several deisgn possibilities
+    - Open a RFC, reaching a decision on the design
+    - Implement the RFC for the unstable feature
+
+
+
+Each PR will still need to be approved by Bevy community members, and would still need to have the same level of quality as other PRs merged.
+
+The PRs for the initial feature, various changes during its finalization and for the stabilization can be opened by different persons.
+
+The feature gate should not be exposed on the main `bevy` crate, but only on the subcrates where it is relevant. It can be enabled as a user by adding dependencies directly on the subcrates and enabling the feature. The tracking issue should list the subcrates impacted.
+
+For organization and discovery, a GitHub project can be used to track the stabilization issues.
+
+## Drawbacks
+
+This will increase complexity in Bevy code.
+
+It will also increase complexity of a contribution that would like to use this process, but it is opt in and not mandatory. Think of it as a fast track for unstable features.
+
+Stabilizing a feature could be harder, as it would mean a review across several PRs, and may not easily work with GitHub review UI. But the stabilization review should be easier than a large PR as there would be actual usage feedback.
+
+An unstable feature accepted in Bevy may stop working due to the merge of another PR. This is the same burden as keeping a PR up to date with Bevy while it's waiting to be merged, but more visible. On the positive side, it would be easier for someone else to jump in and fix it.
+
+## Rationale and alternatives
+
+- Improve the process to get new features in Bevy that would benefit from iterative work.
+- Alternatives:
+    - Continue as it is now, with semi large PRs that bitrot and RFCs that don't often move forward
+    - Work harder to implement a new feature as a third party crate (this is not always possible)
+    - Fork Bevy to merge unnaproved PRs and check how they work / interact
+
+
+## Prior art
```

Comment:
Original article and the discussion here are worth analyzing too.

https://www.reddit.com/r/rust/comments/sy65f7/we_need_to_talk_about_rfcs/